### DISCUSSION TITLE
Option to replace certain Fax messages with Adviser messages

### DISCUSSION THREAD

**Discussion by MandrakeHorse** [CorsixTH/CorsixTH#2661] [Research Room]
### What feature or enhancement would you like to add?

The fax machine will typically "convey important information about emergencies and impending crises that come along during the game." However, there are some fax messages such as New Disease Discovered which aren't as important but must still be opened to be dismissed. One possible enhancement could be to allow users to configure fax messages so that certain messages will be delivered by the Adviser instead (which would require the creation of new strings).

Initially, this would cover the New Disease Discovered messages, since these can become repetitive and are often irrelevant if you've already built a pharmacy or the corresponding clinc. Afterwards, however, we could expand this feature to cover other Miscellaneous Messages (i.e. Emergency/VIP/Epidemic results).

### How would this feature benefit the gameplay of CorsixTH?

More experienced players would be able to personalise which types of fax messages they'd like to receive, reducing the amount of notifications they receive.
More experienced players will be able to receive information via the Adviser, rather than having to manually open and close fax messages.

### Was the feature in the original game?

No

### What version of CorsixTH are you using?

0.68

### Does this feature/enhancement relate to a current problem?

Should be considered in conjunction with #2658 and #530

**Comment by lewri** [comment]
As this is likely going to be an enhancement requiring a lot of definition and discussion I will convert this one to belong in Discussions
